README.md

```diff
@@ -259,20 +267,23 @@ Perform the following steps to install the pre-commit hook:
-
-| Command                                   | Description                                                                                                                           |
-|-------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------|
-| [auth](#using-the-auth-command)           | Authenticates your machine to associate CLI with your Cycode account.                                                                 |
-| [configure](#using-the-configure-command) | Initial command to authenticate your CLI client with Cycode using client ID and client secret.                                        |
-| [ignore](#ignoring-scan-results)          | Ignore a specific value, path or rule ID.                                                                                             |
-| [scan](#running-a-scan)                   | Scan content for secrets/IaC/SCA/SAST violations. You need to specify which scan type: `ci`/`commit_history`/`path`/`repository`/etc. |
-| [report](#report-command)                 | Generate report for SCA SBOM.                                                                                                         |
-| version                                   | Show the version and exit.                                                                                                            |
+| Option                               | Description                                                            |
+|--------------------------------------|------------------------------------------------------------------------|
+| `-v`, `--verbose`                    | Show detailed logs.                                                    |
+| `--no-progress-meter`                | Do not show the progress meter.                                        |
+| `--no-update-notifier`               | Do not check CLI for updates.                                          |
+| `-o`, `--output [text\|json\|table]` | Specify the output (`text`/`json`/`table`). The default is `text`.     |       
+| `--user-agent TEXT`                  | Characteristic JSON object that lets servers identify the application. |
+| `--help`                             | Show options for given command.                                        |
+
+| Command                                   | Description                                                                                                                                  |
+|-------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------|
+| [auth](#using-the-auth-command)           | Authenticates your machine to associate CLI with your Cycode account.                                                                        |
```

Comment:
I know this is existing wording, but a small suggestion since we are making changes here:

```suggestion
| [auth](#using-the-auth-command)           | Authenticates your machine to associate your CLI with your Cycode account.                                                                        |
```

Reply:
ty! fixed